test/multisig/administrator.test.mjs

```diff
@@ -208,21 +214,29 @@ describe(`Testing Administrator`, () => {
 
     it('Should allow to register first proposer', async () => {
       // Before registering
-      const { proposers: proposersBeforeRegister } = await nf3Proposer.getProposers();
-      const fProposersBeforeRegister = proposersBeforeRegister.filter(
-        p => p.thisAddress === nf3Proposer.ethereumAddress,
+      const proposersBeforeRegister = await axios.get(`${optimistApiUrl}/proposer/proposers`);
+      const currentProposerBefore = await axios.get(`${optimistApiUrl}/proposer/current-proposer`);
+
+      const fProposersBeforeRegister = proposersBeforeRegister.data.proposers.filter(
+        p => p.thisAddress !== currentProposerBefore.data.currentProposer,
```

Comment:
Actually, if we look at the assertions, this test will fail in a multiproposer environment. Can you please update L243 and 244 to use [chai.js - include](https://www.chaijs.com/api/bdd/#method_include)?

- `fProposersBeforeRegister` array should not include `ethAddress`
- `fProposersAfterRegister` should include `ethAddress`

Thank you

Reply:
Sorry but I think I got confused. Because we are filtering the array, the assertions should be okay.

There are many ways to write assertions.
An alternative would be to simply demonstrate that the array of proposers includes/or does not include the address - which is what I was suggesting.